apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -738,7 +738,9 @@ namespace MWRender
 
         mViewer->getCamera()->accept(*createIntersectionVisitor(intersector, ignorePlayer, ignoreActors));
 
-        return getIntersectionResult(intersector);
+        RayResult result = getIntersectionResult(intersector);
+        result.mDistanceToFirstIntersection = maxDistance * intersector->getFirstIntersection().ratio;
```

Comment:
mDistanceToFirstIntersection is uninitialized when using the castRay() method. Even if it's not used right now, never a good idea to have uninitialized values flying around.

```getIntersectionResult``` returns the ```RayResult```, it should be initialized there. We can change it to ```result.mRatio``` if that makes it easier.